components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java

```diff
@@ -45,6 +45,12 @@
     @Element(description = "Report generation configurations")
     Map<String, Object> reportGeneration = Collections.emptyMap();
 
+    @Element(description = "Map of roles list")
```

Comment:
Is this description correct?

Reply:
Fixed in 8370d4d40671746b5d46d947c1daad2f0fffd5a5 .